src/workers/folder.ts

```diff
@@ -187,6 +188,9 @@ export const handleFolderModified = (modified: Array<Partial<UserFolder>>): void
 			const folder = folders[val.id];
 			if (folder) {
 				Object.assign(folder, omit(val));
+				if (typeof val.f !== 'undefined') {
+					folder.checked = /#/.test(val.f ?? '');
```

Comment:
this test is used in 2 different ways here and at line 50. Is this wanted? If they do the same (but also if they don't) I would put this test inside a util function because it's not readable as is imo

Reply:
it is wanted, I created a function in utils